packages/core/saved-objects/docs/openapi/bundled.yaml

```diff
@@ -18,6 +18,151 @@ tags:
+                  oneOf:
+                    - type: string
+                    - type: array
+                      items:
+                        type: string
+            examples:
+              exportSavedObjectsRequest:
+                $ref: '#/components/examples/export_objects_request'
+      responses:
+        '200':
+          description: Indicates a successful call.
+          content:
+            multipart/form-data:
+              schema:
+                type: string
+              examples:
+                exportSavedObjectsResponse:
+                  $ref: '#/components/examples/export_objects_response'
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties: true
+      servers:
+        - url: https://localhost:5601
+    servers:
+      - url: https://localhost:5601
+  /api/saved_objects/_import:
+    post:
+      summary: Create sets of Kibana saved objects from a file created by the export API.
+      operationId: importSavedObjects
+      description: |
+        This functionality is in technical preview and may be changed or removed in a future release. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features. Saved objects can be imported only into the same version, a newer minor on the same major, or the next major. Exported saved objects are not backwards compatible and cannot be imported into an older version of Kibana.
+      tags:
+        - saved objects
+      parameters:
+        - $ref: '#/components/parameters/kbn_xsrf'
+        - in: query
+          name: compatibilityMode
+          schema:
+            type: boolean
+          required: false
+          description: |
+            Applies various adjustments to the saved objects that are being imported to maintain compatibility between different Kibana versions. Use this option only if you encounter issues with imported saved objects. NOTE: This option cannot be used with the `createNewCopies` option.
+        - in: query
+          name: createNewCopies
+          schema:
+            type: boolean
+          required: false
+          description: |
+            Creates copies of saved objects, regenerates each object ID, and resets the origin. When used, potential conflict errors are avoided. NOTE: This option cannot be used with the `overwrite` and `compatibilityMode` options.
+        - in: query
+          name: overwrite
+          schema:
+            type: boolean
+          required: false
+          description: |
+            Overwrites saved objects when they already exist. When used, potential conflict errors are automatically resolved by overwriting the destination object. NOTE: This option cannot be used with the `createNewCopies` option.
+      requestBody:
```

Comment:
Is this missing the body param name? It looks like the description of `file`:
<img width="1041" alt="Screenshot 2023-08-23 at 1 02 29 PM" src="https://github.com/elastic/kibana/assets/21210601/1f309307-74c4-4cc4-84e4-720190f20b3c">

Reply:
Thanks — I've fixed this up. Here's an updated screenshot:

<img width="749" alt="Screenshot 2023-08-23 at 5 16 31 PM" src="https://github.com/elastic/kibana/assets/40268737/85c9533a-9a4d-4141-adf3-f33fb5aa9c78">